addons/binding/org.openhab.binding.netatmo/.classpath

```diff
@@ -12,5 +11,6 @@
 	<classpathentry kind="lib" path="lib/json-20131018.jar"/>
 	<classpathentry kind="lib" path="lib/commons-codec-1.8.jar"/>
 	<classpathentry kind="lib" path="lib/gson-2.3.1.jar"/>
```

Comment:
You're right, it looks strange to provide a gson library. Maybe one used library requires a particular version of gson ?